### PR TITLE
Update PlanetScope scripts

### PR DESCRIPTION
**Pull request by zcernigoj**
Updated / added the NDWI scripts
- cleaned up `script.js`which contains "normal" visualization without additional outputs needed for EO Browser statistics and histogram 
- added `eob.js` which contains additional outputs needed for EO Browser statistics and histogram 
- added `raw.js` which just outputs raw NDWI values

Updated `true color` and `false color` scripts
- `script.js` for both now uses `dataMask` to make pixels with no data transparent

### CODE REVIEW

**Review by chorng — CHANGES_REQUESTED** [review]
Add some comments for minor changes.

**Comment by matthew-ballard** [comment]
hi @chorng and @zcernigoj , I just submitted #317 before seeing #308, are you planning to merge this one? or would you be interested in consolidating the two PRs? There is some overlap

**Comment by zcernigoj** [comment]
Hey @matthew-ballard 
I missed the review from @chorng , let me make the changes and merge, and then you can include the updates in your PR.

I guess that would be easiest.

File: planet_scope/ndwi/eob.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      { id: "dataMask", bands: 1 },
+    ]
+  }
+}
+
+function evaluatePixel(samples) {
+  let ndwi = (samples.Green / 3000 - samples.NIR / 3000) / (samples.Green / 3000 + samples.NIR / 3000);
+  const indexVal = samples.dataMask === 1 ? ndwi : NaN;
+
+  let id_default = colorBlend(ndwi,
```
[the remark this answers is not in the document]
Comment (zcernigoj) [reply planet_scope/ndwi/eob.js:21]: Thanks, removed it and added it only in the return statement

File: planet_scope/ndwi/script.js [hunk 2]
```diff
@@ -1,15 +1,30 @@
...
+    output: [
+      { id: "default", bands: 4 },
+    ]
+  }
+}
+
+function evaluatePixel(samples) {
+  let ndwi = (samples.green / 3000 - samples.nir / 3000) / (samples.green / 3000 + samples.nir / 3000);
+
+  let id_default = colorBlend(ndwi,
```
[the remark this answers is not in the document]
Comment (zcernigoj) [reply planet_scope/ndwi/script.js:16]: Thanks, removed it and added it only in the return statement

File: planet_scope/ndwi/script.js [hunk 3]
```diff
@@ -1,15 +1,30 @@
...
+function setup() {
+  return {
+    input: [{ bands: ["green", "nir", "dataMask"] }],
+    output: [
+      { id: "default", bands: 4 },
+    ]
+  }
+}
+
+function evaluatePixel(samples) {
```
[the remark this answers is not in the document]
Comment (zcernigoj) [reply planet_scope/ndwi/script.js]: Thanks, renamed it to `sample`

File: planet_scope/ndwi/script.js [hunk 4]
```diff
@@ -1,15 +1,30 @@
...
+      [1, 0.5, 0, samples.dataMask],
+      [1, 1, 0, samples.dataMask],
+      [0.2, 1, 0.5, samples.dataMask],
+      [0, 0, 1, samples.dataMask],
+      [0, 0, 0.3, samples.dataMask],
+      [0, 0, 0, samples.dataMask],
+    ]
+  );
+
+  return { default: id_default };
```
[the remark this answers is not in the document]
Comment (zcernigoj) [reply planet_scope/ndwi/script.js]: I decided to use `[...id_default, sample.dataMask]`

File: planet_scope/ndwi/raw.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+//VERSION=3
+//NDWI
+
+function setup() {
+  return {
+    input: [{ bands: ["green", "nir", "dataMask"] }],
```
[the remark this answers is not in the document]
Comment (zcernigoj) [reply planet_scope/ndwi/raw.js]: Thanks, removed.

File: planet_scope/ndwi/raw.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    output: [
+      { id: "default", bands: 1, sampleType: "FLOAT32" },
+    ]
+  }
+}
+
+function evaluatePixel(samples) {
```
[the remark this answers is not in the document]
Comment (zcernigoj) [reply planet_scope/ndwi/raw.js]: Thanks, renamed it to `sample`

File: planet_scope/ndwi/raw.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+  let ndwi = index(samples.green, samples.nir);
+  return {default: [ndwi]};
```
[the remark this answers is not in the document]
Comment (zcernigoj) [reply planet_scope/ndwi/raw.js]: Thanks, used the suggested code

File: planet_scope/ndwi/eob.js [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+    output: [
+      { id: "default", bands: 4 },
+      { id: "index", bands: 1, sampleType: "FLOAT32" },
+      { id: "eobrowserStats", bands: 2, sampleType: "FLOAT32" },
+      { id: "dataMask", bands: 1 },
+    ]
+  }
+}
+
+function evaluatePixel(samples) {
```
[the remark this answers is not in the document]
Comment (zcernigoj) [reply planet_scope/ndwi/eob.js]: Thanks, renamed it to `sample`

File: planet_scope/ndwi/eob.js [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+      [1, 1, 0, samples.dataMask],
+      [0.2, 1, 0.5, samples.dataMask],
+      [0, 0, 1, samples.dataMask],
+      [0, 0, 0.3, samples.dataMask],
+      [0, 0, 0, samples.dataMask],
+    ]
+  );
+
+  return {
+    default: id_default,
```
[the remark this answers is not in the document]
Comment (zcernigoj) [reply planet_scope/ndwi/eob.js]: I decided to use [...id_default, sample.dataMask]

File: planet_scope/ndwi/eob.js [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+      { id: "index", bands: 1, sampleType: "FLOAT32" },
+      { id: "eobrowserStats", bands: 2, sampleType: "FLOAT32" },
+      { id: "dataMask", bands: 1 },
+    ]
+  }
+}
+
+function evaluatePixel(samples) {
+  let ndwi = (samples.Green / 3000 - samples.NIR / 3000) / (samples.Green / 3000 + samples.NIR / 3000);
+  const indexVal = samples.dataMask === 1 ? ndwi : NaN;
```
[the remark this answers is not in the document]
Comment (zcernigoj) [reply planet_scope/ndwi/eob.js]: Thanks, added the check for infinite number

**Pull request merged**